src/components/periodic-report/dto/list-periodic-reports.dto.ts

```diff
@@ -25,6 +26,9 @@ export class PeriodicReportListInput extends SortablePaginationInput<
   })
   readonly type?: ReportType;
 
+  readonly startDate?: CalendarDate;
+  readonly endDate?: CalendarDate;
```

Comment:
I'd do it like this
https://github.com/SeedCompany/cord-api-v3/blob/73b7b730ac22152c6b7bbc6f154a71ce2c429958/src/components/project/dto/list-projects.dto.ts#L58-L64
```ts
readonly startDate?: DateFilter;
readonly endDate?: DateFilter;
```
Then your filters can be
```ts
startDate: filter.dateTimeProp(),
endDate: filter.dateTimeProp(),
```

Reply:
I'm good above; I believe this was discussed outside of GH.